### PR TITLE
nlohmann -> rapidjson (zedwallet++)

### PR DESCRIPTION
**Pull request by rixombea**
`ab_add`, `ab_list`, `ab_delete` was tested, no regression.

baby steps for: #714

### CODE REVIEW

File: src/zedwallet++/AddressBook.h [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
+#include "rapidjson/writer.h"
 
 #include <string>
-
 #include <vector>
-
 #include <WalletBackend/WalletBackend.h>
 
-using nlohmann::json;
+using namespace rapidjson;
```
Comment (zpalmtree) [inline src/zedwallet++/AddressBook.h]: I'm not a big fan of `using namespace ...` unless it saves a lot of complexity.

File: src/zedwallet++/AddressBook.h [hunk 2]
```diff
@@ -2,15 +2,15 @@
 // 
 // Please see the included LICENSE file for more information.
 
-#include "json.hpp"
+#include <JsonHelper.h>
+#include "rapidjson/document.h"
+#include "rapidjson/writer.h"
```
Comment (zpalmtree) [inline src/zedwallet++/AddressBook.h]: Do we need to include this here when we're taking a template in `toJSON`?

File: src/zedwallet++/AddressBook.h (hunk 1 above)
Comment (rixombea) [reply src/zedwallet++/AddressBook.h]: removed

File: src/zedwallet++/AddressBook.h (hunk 2 above)
Comment (rixombea) [reply src/zedwallet++/AddressBook.h]: removed

**Pull request merged**